packages/redux-routine/src/is-action.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import { isPlainObject } from 'is-plain-object';
+import type { AnyAction } from 'redux';
+
+/**
+ * Returns true if the given object quacks like an action.
+ *
+ * @param object Object to test
+ *
+ * @return Whether object is an action.
+ */
+export function isAction( object: any ): object is AnyAction {
```

Comment:
Let us use `unknown` instead of `any` here.

```suggestion
export function isAction( object: unknown ): object is Action {
```


We may need to add this to a `types.ts` file in the same directory

```ts
/**
 * Add type guard for isPlainObject until the below PR gets merged:
 *
 * @see https://github.com/jonschlinkert/is-plain-object/pull/29
 */

declare module 'is-plain-object' {
	export function isPlainObject(
		value: unknown
	): value is Record< PropertyKey, unknown >;
}
```